File: DSA/Satyam_Jhinkwan_590012742/Day4/3.cpp
#include <iostream>
#include <vector>
using namespace std;

int main()
{
    int n, val;
    cout << "Enter size of array: ";
    cin >> n;
    vector<int> arr(n);
    cout << "Enter the elements:\n";
    for (auto &x : arr)
        cin >> x;
    cout << "Enter the value : ";
    cin >> val;
    int i = 0;
    for (int j = 0; j < n; j++)
    {
        if (arr[j] != val)
            arr[i++] = arr[j];
    }
    cout << "Length of the new array: " << i;
    return 0;
}

File: DSA/Sandeep_Jha_590014310/week3/day20/3.py
def kth_largest_element(arr, k):
    if not arr:
        return None
    
    if k <= 0 or k > len(arr):
        return None
    
    arr.sort(reverse=True)
    return arr[k-1]

print(kth_largest_element(list(map(int,input("enter the array: ").replace(","," ").split())),int(input("enter the k: "))))